project/profiles/capacity4more/modules/c4m/user/c4m_user_notifications/includes/c4m_user_notifications_handlers.inc

```diff
@@ -501,7 +501,6 @@ function _c4m_user_notifications_notify_content_followers_of_comment($cid, $nid)
   );
 
   $subscribe_options = _c4m_user_notifications_get_skip_entity_access_options();
-  c4m_message_resolve_tokens($message);
```

Comment:
please triple check it doesn't effect and/or not cause new issues with the notifications.